extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/runtime/ResteasyReactiveContextResolver.java

```diff
@@ -0,0 +1,6 @@
+package io.quarkus.rest.client.reactive.runtime;
+
+import jakarta.ws.rs.ext.ContextResolver;
+
+public interface ResteasyReactiveContextResolver<T> extends ContextResolver<T> {
```

Comment:
Let's add some Javadoc to this too just explaining it's functionality

Reply:
Added